scrubbed/main.py

```diff
@@ -75,14 +76,12 @@ def webhook():
 
             logger.debug("sending: \n%s", alert)
 
-            session = requests.Session()
-
-            # Copy headers
-            session.headers.clear()
-            for h in request.headers:
-                session.headers[h] = request.headers.get(h)
-
-            r = session.post(URL, json=alert)
+            r = requests.post(
+                URL,
+                json=alert,
+                headers=request.headers,
```

Comment:
i had a bad feeling when changing this just to make testing easier, but the longer i think about it, the more it seems like the two implementations achieve the exactly same thing under the hood.

Reply:
I moved to session exactly because headers=request.headers didn't work directly

Reply:
yeah, I know... and i'm pretty sure i've done this before as well but i can't seem to remember why it didn't work directly

Reply:
# By using the 'with' statement we are sure the session is closed, thus we
    # avoid leaving sockets open which can trigger a ResourceWarning in some
    # cases, and look like a memory leak in others.
    with sessions.Session() as session:
        return session.request(method=method, url=url, **kwargs)

is whole request function

post function is  return request("post", url, data=data, json=json, **kwargs)

request headers is just plain dict, while Flask request is werzeug.EnvironHeaders inheriting from werkzeug.Headers, closest those classes do is implement __iter__,  so headers=dict(request.headers) ?